index.md

```diff
@@ -224,6 +224,19 @@ modification depends on whether you updated your dependencies in order to fix
 a bug or introduce new functionality. I would usually expect additional code
 for the latter instance, in which case it's obviously a minor level increment.
 
+#### Special note for JVM or similar environments
+
+In such environments where only one version of a library could 
+be present in a running system, the transitive dependencies effectively become
+part of the public library interface and thus the version of the user library SHOULD be 
```

Comment:
Why to not reword section in order to use MUST (instead of SHOULD). 

I suggest to add "[...] If any potential client environment allows only one version of a library be present in running system, [...] user library MUST be [...]"

The effort is to clarify when you MUST do something. Not simply recommending with SHOULD.

Reply:
Updated